model/index.ts

```diff
@@ -0,0 +1,103 @@
+export {};
```

Comment:
?

Reply:
Приходилось добавлять изначально, т.к. иначе тайпскрипт жаловался 'server.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.ts(1208).